frontend/src/routes.tsx

```diff
@@ -77,16 +77,25 @@ export default function AppRoutes () {
     ]);
 
     useMemo(() => {
-        dispatch(getConfiguration({configScope: 'global'}));
+        let configFetched = false;
+
+        async function fetchAppConfig () {
+            if (!configFetched) {
```

Comment:
Won't this always be false since it is reset when useMemo() is called?

Reply:
It's some React magic.

[Here's the documentation](https://react.dev/learn/synchronizing-with-effects#fetching-data), the relevant section is:
>In development, you will see two fetches in the Network tab. There is nothing wrong with that. With the approach above, the first Effect will immediately get cleaned up so its copy of the ignore variable will be set to true. So even though there is an extra request, it won’t affect the state thanks to the if (!ignore) check.